app/src/main/java/com/github/fribourgsdp/radio/database/FirestoreDatabase.kt

```diff
@@ -465,9 +465,9 @@ class FirestoreDatabase(var refMake: FirestoreRef) : Database {
             .update(updatesMap)
     }
 
-    override fun updateCurrentSongOfGame(id: Long, songName: String): Task<Void> {
+    override fun updateCurrentSongOfGame(id: Long, songName: String, incrementBy: Long): Task<Void> {
         val roundDeadline = Date()
-        roundDeadline.time += ROUND_TIME_IN_MILLIS
+        roundDeadline.time += incrementBy
```

Comment:
Did you make sure to use milliseconds ? Because I saw that `DEFAULT_GAME_DURATION` is `45L`.

Reply:
Good catch!

Reply:
Faster than Lightning McQueen 😂  *Ka Chow!*